src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/MockSentencingResource.kt

```diff
@@ -21,19 +23,40 @@ class MockSentencingResource {
   private companion object {
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
+
   @PreAuthorize("hasRole('ROLE_SENTENCING_SYNC')")
   @PostMapping("/migration/sentencing/adjustments")
   @Operation(hidden = true)
-  suspend fun createAdjustment(
-    @RequestBody @Valid createAdjustmentRequest: MockCreateAdjustmentRequest
+  suspend fun createAdjustmentForMigration(
+    @RequestBody @Valid adjustmentRequest: MockAdjustmentRequest
+  ): MockCreateAdjustmentResponse {
+    val id = Random.nextLong().toString()
+    log.info("Created adjustment for migration with id $id for booking ${adjustmentRequest.bookingId}. Request was $adjustmentRequest")
+    return MockCreateAdjustmentResponse(id)
+  }
+
+  @PostMapping("/synchronisation/sentencing/adjustments")
+  @Operation(hidden = true)
+  suspend fun createAdjustmentForSynchronisation(
```

Comment:
I have deliberately not add roles to these mock endpoints until I understand the role situation more